crates/sui/src/client_commands.rs

```diff
@@ -1598,6 +1609,9 @@ impl Display for SuiClientCommandResult {
             SuiClientCommandResult::SerializeTransferSui(data) => {
                 writeln!(writer, "Raw tx_bytes to execute: {}", data)?;
             }
+            SuiClientCommandResult::SerializePublish(data) => {
+                writeln!(writer, "Raw tx_bytes to execute: {}", data)?;
```

Comment:
Would it be easier to use this if we didn't prefix the data with the 'Raw tx_bytes to execute' label?  If you want, you could print the information message to stderr instead?  Avoids that step of stripping this stuff out using `sed`.

Reply:
Also, if we're going to have lots of these, it would be good to just have a single `Serialized` command result type that they all return.  And, not for this PR, but we should consider splitting up all the client commands that send transactions so that they have a standard interface that produces transaction data at the end, which we can decide to serialize or execute.

Reply:
> Would it be easier to use this if we didn't prefix the data with the 'Raw tx_bytes to execute' label?

i would like to refactor the output formats in another PR (i'd like to change how `SerializeTransferSui` also formats output, and I'd like to make it easier overall to grab info from the cli (e.g., grab the keys from `keytool list`, etc)